src/assets/toolkit/styles/templates/_print.scss

```diff
@@ -74,11 +74,22 @@
 .hide-for-print {
   @media print {
     display: none;
-  }  
+  }
 }
 
 .show-for-print {
   @media print {
     display: block;
-  }  
+  }
+}
+
+@media print {
+  [ng\:cloak], [ng-cloak], [data-ng-cloak], [x-ng-cloak], .ng-cloak, .x-ng-cloak, .ng-hide:not(.ng-hide-animate) {
+   display: block !important;
```

Comment:
@chrisdolendo noticing some funkiness with the More/Less links and this line which is telling it to show `.ng-hide` elements, and it's also showing the ellipsis. I wonder if we shouldn't just fork the ng-text-truncate to make it more print-friendly i.e. sticking in `show/hide-for-print` classes as necessary. We're already even using a fork of the original project. The project itself is really just one pretty small JS file.